diff -ruN /var/folders/ry/fwbrlv8s0vj2h28gvf8w39hr0000gn/T/esc_pos_utils_plus/lib/src/generator.dart /Users/ikbalmoh/.pub-cache/hosted/pub.dev/esc_pos_utils_plus-2.0.4/lib/src/generator.dart
--- /var/folders/ry/fwbrlv8s0vj2h28gvf8w39hr0000gn/T/esc_pos_utils_plus/lib/src/generator.dart	2025-02-04 15:09:52
+++ /Users/ikbalmoh/.pub-cache/hosted/pub.dev/esc_pos_utils_plus-2.0.4/lib/src/generator.dart	2025-02-10 10:54:11
@@ -413,7 +413,7 @@
   /// [mode] is used to define the full or partial cut (if supported by the printer)
   List<int> cut({PosCutMode mode = PosCutMode.full}) {
     List<int> bytes = [];
-    bytes += emptyLines(5);
     if (mode == PosCutMode.partial) {
       bytes += cCutPart.codeUnits;
     } else {
